include/SPTDataLoader/module.modulemap

```diff
@@ -0,0 +1,19 @@
+module SPTDataLoader {
+    umbrella header "SPTDataLoader.h"
+
+    header "SPTDataLoaderAuthoriser.h"
```

Comment:
Surely listing these as well as the `umbrella header` clause is redundant?

Reply:
I dunno! I'm going by some examples I've seen elsewhere.

Reply:
Figured it would be best to be as complete as possible.

Reply:
I really don’t think so, this is effectively specifying headers twice which is at best unnecessary complexity